src/ray/core_worker/core_worker.cc

```diff
@@ -4651,7 +4649,7 @@ void CoreWorker::PlasmaCallback(SetResultCallback success,
       GetAsync(object_id, success, py_future);
     };
 
-    async_plasma_callbacks_[object_id].push_back(plasma_arrived_callback);
+    async_plasma_callbacks_[object_id].emplace_back(std::move(plasma_arrived_callback));
```

Comment:
nit: I would prefer to `[std::move(object_id)]`

Reply:
objectId is trivial copy iiuc

Reply:
Yeah, see my argument above.

Reply:
object_id is actually used right below